layouts/default.vue

```diff
@@ -2,6 +2,7 @@
   <v-app ref="app">
     <backdrop />
     <v-navigation-drawer
```

Comment:
It'd probably be interesting to investigate how feasible it'd be to remove the transition for this when moving to the full screen player.

The current animation when hiding the drawer looks a bit weird. Not really blocking though.